example/usps_api/server.cc

```diff
@@ -0,0 +1,43 @@
+#include <string>
+#include <iostream>
+
+#include <grpc/grpc.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+#include <grpcpp/security/server_credentials.h>
+
+/* https://grpc.io/docs/languages/cpp/basics/
+ * Runs server using server builder
+ */ 
+void RunServer(std::string host, std::string port) {
+  std::string server_address = host + ":" + port;
```

Comment:
Use absl::StrCat then you can take an int as the port: https://github.com/abseil/abseil-cpp/blob/master/absl/strings/str_cat.h.

Reply:
Did something similar by using uin16_t as the flag type